Makefile

```diff
@@ -10,3 +10,8 @@ init:
 
 test:
 	$(MAKE) -C tests
+
+syntax:
```

Comment:
Should it be `.PHONY` here to avoid caching?

Reply:
Maybe better `verify`? So a default execution would init the framework and check the syntax and the tests

Reply:
I would make it so that we don't need to change the way of invoking this from jenkinsfile-runner.

Reply:
Good point @varyvol ! @oleg-nenashev to avoid caching any result of tests or shellcheck, I'll remove the directories in the `init` goal